### PR TITLE
On macOS, display the installer version in the window title

### PR DESCRIPTION
**Pull request by hoechenberger**

### CODE REVIEW

**Comment by kenodegard** [comment]
@conda-bot check

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,4 +1,15 @@
 .. current developments
+
+XXXX-XX-XX   X.X.X:
+===================
+
+Enhancements:
+-------------
+
+* The graphical macOS installer now also displays the version number of the
+  software. (#536)
```
Comment (hoechenberger) [inline CHANGELOG.rst:10]:
```suggestion
  software in the window title. (#536)
```

**Comment by larsoner** [comment]
The bot and CI say "CLA check — CLA signed, thank you!", so this is okay to merge at this point, right @kenodegard ? I can just do it in the future if that's the case, but want to double check this first time. I haven't worked with CLAs much...

**Comment by kenodegard** [comment]
@larsoner Yes, you are free to finish reviewing/merging.

FYI, the process in place is we get the signed CLA form, we add the user's GitHub login to https://github.com/conda/infra/blob/main/.clabot, and then retrigger the CLA check (by reopening the PR, adding more commits, or commenting `@conda-bot check`).

**Comment by larsoner** [comment]
> FYI, the process in place is

Nice, thanks for the info!

This seems like a pretty straightforward enhancement, we can always add an option to disable it if people want in the future (but I'm assuming YAGNI for now), thanks @hoechenberger !

**Pull request merged**